site/content/integrate/webhooks/incoming/_index.md

```diff
@@ -2,5 +2,193 @@
 title: Incoming webhooks
 description: Incoming webhooks
 weight: 10
+aliases:
+  - /integrate/webhooks/incoming/incoming-webhooks/
+  - /integrate/webhooks/incoming/using-incoming-webhooks/
 ---
-Incoming webhooks let you POST some data to a Mattermost endpoint to create a message in a channel.
+## Create an incoming webhook
+
+Let's learn how to create a simple incoming webhook that posts the following message to Mattermost.
+
+![image](incoming_webhooks_create_simple.png)
+
+1. First, go to **Product menu > Integrations > Incoming Webhook**.
+    - If you don't have the **Integrations** option in your Main Menu, incoming webhooks may not be enabled on your Mattermost server or may be disabled for non-admins. They can be enabled by a System Admin from **System Console > Integrations > Integration Management**. Then continue with the steps below.
+2. Select **Add Incoming Webhook** and add name and description for the webhook. The description can be up to 500 characters.
```

Comment:
```suggestion
2. Select **Add Incoming Webhook** and a add name and description for the webhook. The description can be up to 500 characters.
```

Reply:
It looks like the `a` should come after the `add`.

Reply:
Thanks! Suggestion updated!